### DISCUSSION TITLE
What is the correct way to have Terminal Preview launch in Quake Mode at Windows boot?

### DISCUSSION THREAD

**Discussion by Skyyblaze** [microsoft/terminal#17256] [Q&A]
I always had Terminal Preview set to restore the last session which was a Quake Mode window for me and then autostart with Windows. However after a recent update it launches in a normal window instead. I presume this is a bug but I would still like to know what the correct way would be to have Terminal Preview autostart with Windows in Quake Mode. I'm using version 1.21.1272.0.

Thanks for help in advance!

**Comment by zadjii-msft** [comment]
Hmm. I think this _should_ work... it's at least working here for me in the non-autostart situation. There's a file called `state.json` that lives next to your [settings.json file](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#where-can-i-find-the-settings-file). Could you share that with us/?

**Reply by Skyyblaze** [reply]
Thanks for trying to help @zadjii-msft! Sure I can share that:

{
	"dismissedMessages" : 
	[
		"deprecated",
		"closeOnExitInfo"
	],
	"generatedProfiles" : 
	[
		"{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
		"{2c4de342-38b7-51cf-b940-2309a097f518}",
		"{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
		"{b453ae62-4e3d-5e58-b989-0a998ec441b8}",
		"{574e775e-4f2a-5b96-ac1e-a2962a402336}"
	],
	"persistedWindowLayouts" : 
	[
		{
			"initialPosition" : "0,0",
			"initialSize" : 
			{
				"height" : 656,
				"width" : 2560
			},
			"launchMode" : "default",
			"tabLayout" : []
		}
	],
	"settingsHash" : "3cbfaee24dda78a9-01daa209a001829c"
}

I hope it works this way,
